nixos/modules/services/databases/postgresql.nix

```diff
@@ -187,6 +192,16 @@ in
               '';
             };
 
+            ensureDBOwnership = mkOption {
+              type = types.bool;
+              default = false;
```

Comment:
Would it make sense to default this to true?

Reply:
I fear this may explode for people having user != database?

Reply:
to be fair, it would explode, for users that don't have a corresponding DB.

Reply:
Let's make it default in a later PR if that's okay because there's already too many things to do and I don't want to drop the ball on this